frontend/app/js/script.js

```diff
@@ -29,11 +29,18 @@ $(function() {
     $("a[data-action='enlarge']",codeDiv).show();
   });
 
-  $('form input[type=submit], form button[type=submit]').on('click', function() {
-    var $this = $(this);
+  $('form').on('submit', function() {
+    var $this = $(this).find('button');
```

Comment:
This no longer catches `input[type="submit"]`. Don't know if there are any. Maybe it's just buttons.